dev/com.ibm.ws.jdbc.4.3/src/com/ibm/ws/rsadapter/jdbc/v43/WSJdbc43PreparedStatement.java

```diff
@@ -46,27 +48,63 @@ public WSJdbc43PreparedStatement(PreparedStatement pstmtImplObject, WSJdbcConnec
     public String enquoteLiteral(String val) throws SQLException {
         // KEEP CODE IN SYNC: This method is duplicated in WSJdbc43Statement, WSJdbc43PreparedStatement,
         // and WSJdbc43CallableStatement because multiple inheritance isn't allowed.
-        throw new UnsupportedOperationException();
+        try {
+            return pstmtImpl.enquoteLiteral(val);
+        } catch (SQLException ex) {
+            FFDCFilter.processException(ex,
+                                        "com.ibm.ws.rsadapter.jdbc.v43.WSJdbc43PreparedStatement.enquoteLiteral(String)", "55", this);
+            throw WSJdbcUtil.mapException(this, ex);
+        } catch (NullPointerException nullX) {
+            // No FFDC code needed; we might be closed.
+            throw runtimeXIfNotClosed(nullX);
+        }
     }
 
     @Override
     public String enquoteIdentifier(String identifier, boolean alwaysQuote) throws SQLException {
         // KEEP CODE IN SYNC: This method is duplicated in WSJdbc43Statement, WSJdbc43PreparedStatement,
         // and WSJdbc43CallableStatement because multiple inheritance isn't allowed.
-        throw new UnsupportedOperationException();
+        try {
+            return pstmtImpl.enquoteIdentifier(identifier, alwaysQuote);
+        } catch (SQLException ex) {
+            FFDCFilter.processException(ex,
+                                        "com.ibm.ws.rsadapter.jdbc.v43.WSJdbc43PreparedStatement.enquoteIdentifier(String, String)", "71", this);
```

Comment:
Same as previous comment.  I also wonder if it is even valuable to record FFDC for simple operations that fail in the JDBC driver like this.  Liberty never invokes these of its own choosing. They are only used when the application invokes them, and any exception goes right back to the application which is probably good enough.

Reply:
I'm fine with not logging an FFDC, I was just going off what we did for JDBC 4.1/4.2.